filament/backend/test/Lifetimes.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Lifetimes.h"
+
+#include "backend/Platform.h"
+#include "private/backend/DriverApi.h"
+
+RenderFrame::RenderFrame(filament::backend::DriverApi& api): api_(api) {
+    api_.beginFrame(0, 0, 0);
+}
+
+RenderFrame::~RenderFrame() {
+    api_.endFrame(0);
+}
+
+Cleanup::Cleanup(const test::BackendTest& testFixture) : testFixture_(testFixture) {}
+
+Cleanup::~Cleanup() {
+    auto& api = testFixture_.getDriverApi();
```

Comment:
Will this compile? `testFixture` is marked const, but the `getDriverApi()` method is non-const.

Reply:
Whoops, this change was sliced off of a larger branch and that snuck through somehow sorry. That's what I get for trying to rush it out on a friday evening.